test/main_module_spec.rb

```diff
@@ -0,0 +1,16 @@
+# To change this license header, choose License Headers in Project Properties.
+# To change this template file, choose Tools | Templates
+# and open the template in the editor.
+
+require 'main_dialog'
+
+describe MainDialog do
```

Comment:
this won't work. require needs to use `y2hello_world/maindialog` or helloworld/... if for current name and also describe need to use namespace Y2HelloWorld::MainDialog

Reply:
updated in commit